BuildResidentialHPXML/measure.rb

```diff
@@ -3081,8 +3048,8 @@ def argument_errors(args)
     error = (args[:geometry_unit_type] == HPXML::ResidentialTypeSFD) && (args[:geometry_unit_left_wall_is_adiabatic] || args[:geometry_unit_right_wall_is_adiabatic] || args[:geometry_unit_front_wall_is_adiabatic] || args[:geometry_unit_back_wall_is_adiabatic] || (args[:geometry_attic_type] == HPXML::AtticTypeBelowApartment) || (args[:geometry_foundation_type] == HPXML::FoundationTypeAboveApartment))
     errors << 'No adiabatic surfaces can be applied to single-family detached homes.' if error
 
-    error = (args[:geometry_unit_type] == HPXML::ResidentialTypeApartment) && [HPXML::AtticTypeVented, HPXML::AtticTypeUnvented, HPXML::AtticTypeConditioned].include?(args[:geometry_attic_type])
-    errors << 'Apartment units can only have a flat roof or be below another apartment unit.' if error
+    error = (args[:geometry_unit_type] == HPXML::ResidentialTypeApartment) && (args[:geometry_attic_type] == HPXML::AtticTypeConditioned)
+    errors << 'Conditioned attic type for apartment units is not currently supported.' if error
```

Comment:
Can we call this out in the description of the "geometry_attic_type" argument too? Something like `"#{HPXML::AtticTypeConditioned} is not allowed for apartment units."`

And can you audit these error messages and see if similar language should be added for other arguments? For example, conditioned basement for an apartment unit.